src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private CinemaHall cinemaHall;
```

Comment:
You could edit the column name hereinafter in SQL style:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "cinema_hall_id")
    private CinemaHall cinemaHall;
```